squirrel_safety/launch/squirrel_safety.launch

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<launch>
+  <node name="squirrel_safety_node" pkg="squirrel_safety" type="squirrel_safety_node.py" output="screen" respawn="true">
+    <param name="airskin_topic" value="/arm_bumper" type="string"/>
```

Comment:
Ah, does it add a namespace in the launch file? Because in the node it's just called arm_bumper.